squashfs-mount.c

```diff
@@ -17,6 +17,9 @@
 
 #include <libmount/libmount.h>
 
+#define ENV_MOUNT_FILE "SLURM_UENV_MOUNT_FILE"
```

Comment:
These are not SLURM variables - the main consumer of them is intended to be slurm, but other tools could use them.
`"UENV_MOUNT_FILE"` and `"UENV_MOUNT_POINT"` might be less surprising.

Reply:
Thanks. I've removed the `SLURM_` prefix.